### ISSUE TITLE
Fingerprints: HTML not sanitized in backend

### ISSUE THREAD

**Issue by wetneb** [dissemin/dissemin#192]
proaixy does not sanitize titles in fingerprints, this reduces the chances of matches.
plan a fingerprint re-run together with other data-heavy operations

**Comment by RaitoBezarius** [comment]
Where should we sanitize the title then?

**Comment by wetneb** [comment]
that's actually in proaixy, not in this repo. closing this.

**Comment by RaitoBezarius** [comment]
If you open another issue related to it on proaixy bug tracker, can you put a link so we can relate it, @wetneb ?

**Comment by wetneb** [comment]
I am actually thinking about dropping fingerprints in proaixy, so that we compute them only once, in dissemin. Otherwise, that means duplicating fingerprint computation code, and when they start to differ (as it is the case), it's bad.
